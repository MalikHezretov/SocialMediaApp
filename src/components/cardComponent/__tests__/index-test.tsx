import 'react-native';
import React from 'react';
import CardComponent from '..';
import renderer from 'react-test-renderer';

const image = 'https://picsum.photos/id/237/200/300';

it('renders CardComponent correctly', () => {
  const testRenderer = renderer.create(<CardComponent imageUrl={image} />);
  const testInstance = testRenderer.root;

  expect(testInstance.findByType(CardComponent).props.imageUrl).toBe(image);
  expect(testRenderer).toMatchSnapshot();
});
