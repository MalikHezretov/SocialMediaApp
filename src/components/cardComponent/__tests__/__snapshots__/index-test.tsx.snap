// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardComponent correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 16,
      "paddingBottom": 0,
      "paddingTop": 8,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://picsum.photos/id/237/200/300",
      }
    }
    style={
      Object {
        "borderRadius": 2,
        "height": 200,
      }
    }
  />
</View>
`;
